joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_link_link_01
  pid: {p: 800.0, i: 10, d: 2.0}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_01_link_02
  pid: {p: 10.0, i: 1, d: 4.0}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_02_link_03
  pid: {p: 50.0, i: 10, d: 70.0}
joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_03_link_04
  pid: {p: 5, i: 0, d: 0.0010}
